onyo/lib/commands.py

```diff
@@ -401,16 +402,21 @@ def onyo_edit(inventory: Inventory,
     paths
         Paths of assets to edit.
     message
-        A custom commit message.
+        Commit message to append to the auto-generated message.
+    auto_message
+        If `True`, do auto-generate a commit message subject line.
```

Comment:
Agree on this wording being more concise. Otherwise: It is documenting a python function for programmers who want to use it. So, I find the argument a bit odd.

However, will do.